python/kiss_icp/kiss_icp.py

```diff
@@ -41,9 +41,16 @@ def __init__(self, config: KISSConfig):
         self.preprocess = get_preprocessor(self.config)
         self.prev_frame_delta = 1
 
-    def register_frame(self, frame, timestamps, frame_delta=1):
+    def register_frame(self, frame, timestamps, frame_delta = 1):
```

Comment:
[nit] python style actually don't favor leaving spaces on parameter assignment (check [PEP8](https://peps.python.org/pep-0008/#other-recommendations))
```suggestion
    def register_frame(self, frame, timestamps, frame_delta=1):
```